ceph.spec.in

```diff
@@ -266,12 +266,7 @@ BuildRequires:  libzbd-devel
 %if 0%{with jaeger}
 BuildRequires:  bison
 BuildRequires:  flex
-%if 0%{?fedora} || 0%{?rhel}
```

Comment:
could you please reconsider the comment of https://github.com/ceph/ceph/pull/38783/commits/e277cf3ee29063311437d372d1f54bf805a7341e#r664177718
 
also, "cleanup" is a little bit vague. it seems it a must have to enable the opensuse build.

Reply:
> also, "cleanup" is a little bit vague. it seems it a must have to enable the opensuse build.

It is the opposite, since we dropped building for opensuse, we don't need to find pkg's specific to it. From commit-msg:
```
we dropped support for opensuse, which had json.hpp needed for jaeger provided
    by nlohmann_json-devel while centos/RHEL based system packed json.hpp with
    jason-devel pkg. This change will remove suse bases jaeger dependencies from
    ceph.spec
```
https://github.com/ceph/ceph/pull/38783/commits/a68f05e23478458d6fcc7defbff23cfc3ddcf471